common/src/supabase.schema.ts

```diff
@@ -0,0 +1,1113 @@
+export type Json =
```

Comment:
I don't think we need this copy of the schema here, right? You can delete this file.